test/integration/user_can_view_past_order_test.rb

```diff
@@ -1,8 +1,11 @@
 class UserCanViewPastOrderTest < ActionDispatch::IntegrationTest
   test "past order view" do
+    skip
     user = User.create(username: "Sekhar",
                        password: "password")
 
+    # TODO visit ?????????????
```

Comment:
Annotation keywords like `TODO` should be all upper case, followed by a colon, and a space, then a note describing the problem.